com.unity.robotics.urdf-importer/Runtime/AssetHandlers/UrdfAssetPathHandler.cs

```diff
@@ -87,27 +89,33 @@ public static string GetFullAssetPath(string relativePath)
 
         public static string GetRelativeAssetPathFromUrdfPath(string urdfPath, bool convertToPrefab=true)
         {
-            if (!urdfPath.StartsWith(@"package://"))
+            string path;
+            if (!urdfPath.StartsWith(@"file://") && !urdfPath.StartsWith(@"package://"))
             {
-               Debug.LogWarning(@$"{urdfPath} is not a valid URDF package file path. Path should start with package://, and URDF file should be in the directory root.");
                if (urdfPath.Substring(0, 3) == "../")
-               {
-                   Debug.LogWarning("Attempting to replace file path's starting instance of `../` with standard package notation `package://` to prevent manual path traversal at root of directory!");
+                {
+                   UnityEngine.Debug.LogWarning("Attempting to replace file path's starting instance of `../` with standard package notation `package://` to prevent manual path traversal at root of directory!");
                    urdfPath = $@"package://{urdfPath.Substring(3)}";
-               }
-               else
-               {
+                }
+                else
+                {
                    return null;
-               }
+                }
             }
-            string path;
+            // loading assets relative path from ROS/ROS2 package.
             if (urdfPath.StartsWith(@"package://"))
             {
                 path = urdfPath.Substring(10).SetSeparatorChar();
             }
+            // loading assets from file:// type URI.
+            if (urdfPath.StartsWith(@"file://"))
+            {
+                path = urdfPath.Substring(7).SetSeparatorChar();
+                return path;
+            }
             else
             {
-                path = urdfPath.SetSeparatorChar();
+                throw new Exception(urdfPath + " is not supported URI format.");
```

Comment:
This change looks like it would break code execution for cases that previously worked. Why does the function return early in the case of a `file://` prefix? What would happen in the case of a `file://` prefix where `convertToPrefab` is true?

Reply:
Ah, I understand.
Sorry, I missed logic with convertToPrefab.

Reply:
Thank you!
I missed logic with convertToPrefab variable.

Reply:
In file:// URI, it is unnecessary for us to resolve from package path. 
Because they are already resolved.
So I early return.

Reply:
I changed my logic in https://github.com/Unity-Technologies/URDF-Importer/pull/147/commits/12a9204317165b9d22ccabc3c2e86727cc63f517

Please check.